tests/components/peblar/test_select.py

```diff
@@ -33,3 +51,115 @@ async def test_entities(
     )
     for entity_entry in entity_entries:
         assert entity_entry.device_id == device_entry.id
+
+
+@pytest.mark.usefixtures("entity_registry_enabled_by_default")
+async def test_select(
```

Comment:
Can we split this test into several tests?

Reply:
Sure [549507b](https://github.com/home-assistant/core/pull/133804/commits/549507bc887760f0eb19a80338f5b498c7f0a506)